.bazelrc

```diff
@@ -130,3 +138,9 @@ test:strace --run_under="bash -c 'if command -v strace >/dev/null && strace -qq
 # [Linux] Uncomment this line (or use --config) to preload libSegFault.so if available, to print a stack trace on aborts and segfault. (Note: This doesn't always work.)
 #test:linux --config=segfault
 test:segfault --run_under="bash -c 'unset GREP_OPTIONS && if ! grep -q -o Microsoft /proc/version 2>/dev/null; then libs=\"$(command -v ldconfig >/dev/null && ldconfig -p | grep -F -o -e \"libSegFault.so\" | uniq | tr \"\\\\n\" :)\" && if [ -n \"${libs%:}\" ]; then export SEGFAULT_SIGNALS=\"abrt segv\" LD_PRELOAD=\"${libs}${LD_PRELOAD-}\"; fi; fi && \"$@\"' -"
+
+# Debug build:
+build:debug -c dbg
+build:debug --javacopt="-g"
```

Comment:
Maybe disabling this for now? I think we have no java tests we want to test now + this will probably make the wheel size even bigger?